### PR TITLE
Document gotthard

### PR DESCRIPTION
**Pull request by feikesteenbergen**
Adresses issue #1

### CODE REVIEW

**Comment by avaczi** [comment]
:+1:

File: README.rst [hunk 1]
```diff
@@ -1,6 +1,12 @@
 ========
 Gotthard
 ========
+Gotthard is a command line tool to simplify connecting to a PostgreSQL instance via a bastion_ host.
+This tool is created in the context of zalando-stups_, however it does not depend on it.
```
Comment (alexeyklyukin) [inline README.rst]: I'm not sure this claim is valid, given that we are using piu and piu configuration file.

Comment (feikesteenbergen) [inline README.rst]: I can weaken it a bit. However, piu is only needed if you specify a reason.

Something like "however it could be used without it".

**Comment by feikesteenbergen** [comment]
👍

**Comment by avaczi** [comment]
:+1:

**Pull request merged**